app/utils/validators/forms/FieldValidator.scala

```diff
@@ -84,8 +84,8 @@ object FieldValidator {
 
   val startsWithCapitalLetter: String => Boolean = (input: String) => input.headOption.exists(_.isUpper)
 
-  val isContainedIn: Iterable[String] => String => Boolean =
-    (iterable: Iterable[String]) => (input: String) => iterable.exists(_ == input)
+  def isContainedIn[T](iterable: Iterable[T]): T => Boolean =
```

Comment:
Can we add some test that will cover different type? Actually we have tests just for string. It'll be good to have something for some different this that obvious type.

Reply:
If it works for `String` surely it'd work for other types? it'd just be testing `iterable.exists(..)` works?

Reply:
Actually there already is in the AdditionalDeclarationType form tests

Reply:
As it first verifies what was entered was a valid enum value, then converts it to the enum, then verifies what was entered was a valid enum value for the options available.

Probably easier to look at the form mapping